src/styles/index.scss

```diff
@@ -1,3 +1,16 @@
+@import './utils/variables';
+@import './blocks/card';
+@import './blocks/stars';
+@import './blocks/header';
+
+html {
+  margin: 0;
+  padding: 0;
+  font-family: $font-family-primary;
+  font-weight: $font-weight-regular;
+  font-style: normal;
+}
+
 body {
   margin: 0;
 }
```

Comment:
The 'margin: 0;' property is set for both 'html' and 'body' selectors which is redundant. It's a common practice to apply 'margin: 0;' to the 'body' only, as the 'html' element doesn't have default margins.